casepro/cases/models.py

```diff
@@ -435,6 +490,7 @@ def as_json(self, full=True):
             return {
                 'id': self.pk,
                 'assignee': self.assignee.as_json(full=False),
+                'user_assignee': self.user_assignee.as_json(full=False) if self.user_assignee else None,
```

Comment:
Ah yes, the messages views. I'll add the template changes needed to show the assigned user to the various message views, it shouldn't be too big a change. Then I'll prefetch this as well.